docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -288,8 +288,6 @@ Next, the setup downloads a Teleport agent for your platform and runs `teleport
 pointed at the current cluster. Once that is done, Connect My Computer starts the agent and waits
 for it to show up in the cluster as an SSH node.
 
-![Connect My Computer setup](../../img/use-teleport/connect-my-computer-setup@2x.png)
-
```

Comment:
If we are going to restore previous versions of Connect My Computer images, can we bring back this one as well?

Reply:
Done!

Reply:
I see it's removed still. 🫥

Reply:
When I open the "Files changed" tab in this PR, it tells me that [`docs/img/use-teleport/connect-my-computer-setup@2x.png` has been removed](https://github.com/gravitational/teleport/pull/48824/files#diff-f74b32aafbd54623636487e75fba29e77023075a638ce303242a7076de5aa074) and that [it has been removed from `teleport-connect.mdx` as well](https://github.com/gravitational/teleport/pull/48824/files#diff-f9d1425ebeb5497eec68d311c2a20b5fe1616ea10fe943863cd80a452d2f75d0).

Reply:
Apologies @ravicious , I was looking at the wrong 'connect-my-computer-x' image. Restored:

https://github.com/gravitational/teleport/pull/48824/commits/f935faca177ed78c66b0177b617db1fd8b4fa354